webapp/src/main/java/com/box/l10n/mojito/service/delta/PushPullRunCleanupJob.java

```diff
@@ -0,0 +1,71 @@
+package com.box.l10n.mojito.service.delta;
+
+import com.box.l10n.mojito.service.assetExtraction.AssetExtractionCleanupJob;
+import org.quartz.DisallowConcurrentExecution;
+import org.quartz.Job;
+import org.quartz.JobDetail;
+import org.quartz.JobExecutionContext;
+import org.quartz.JobExecutionException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.scheduling.quartz.CronTriggerFactoryBean;
+import org.springframework.scheduling.quartz.JobDetailFactoryBean;
+import org.springframework.stereotype.Component;
+
+import java.time.Duration;
+
+/**
+ * @author garion
+ */
+@Profile("!disablescheduling")
+@Configuration
+@Component
+@DisallowConcurrentExecution
+@ConditionalOnProperty(value = "l10n.PushPullRun.cleanup-job.enabled", havingValue = "true")
```

Comment:
probably my bad when i suggested the name. I think other job use `-` like `l10n.asset-extraction.cleanup-job.enabled`. Maybe we can keep it consistent. Merging as it anyway